### PR TITLE
refactor: ParsedMessage

### PR DESCRIPTION
**Pull request by harikrishnakanchi**
- Refactor getMapping method. Removed schema mapping

### CODE REVIEW

File: src/main/java/io/odpf/depot/message/proto/ProtoOdpfParsedMessage.java [hunk 1]
```diff
@@ -51,92 +49,47 @@ public void validate(OdpfSinkConfig config) {
...
-            String columnName = getNestedColumnName((Properties) value);
-            Object fieldValue = getMappings((DynamicMessage) field, (Properties) value);
-            return new Tuple<>(columnName, fieldValue);
-        } catch (Exception e) {
-            log.error("Exception::Handling nested field failure: {}", e.getMessage());
-            throw e;
+            return getMappings((DynamicMessage) value);
         }
+        ProtoField field = ProtoFieldFactory.getField(fd, value);
+        return field.getValue();
```
Comment (lavkesh) [inline src/main/java/io/odpf/depot/message/proto/ProtoOdpfParsedMessage.java:66]: Let's add the floating point check too.

Comment (harikrishnakanchi) [reply src/main/java/io/odpf/depot/message/proto/ProtoOdpfParsedMessage.java:72]: Added this

File: src/main/java/io/odpf/depot/message/proto/ProtoOdpfParsedMessage.java [hunk 2]
```diff
@@ -52,50 +52,24 @@ public void validate(OdpfSinkConfig config) {
...
+        return getMappings(dynamicMessage);
     }
 
-    @SuppressWarnings("unchecked")
-    private Map<String, Object> getMappings(DynamicMessage message, Properties columnMapping) {
-        if (message == null || columnMapping == null || columnMapping.isEmpty()) {
-            return new HashMap<>();
+    private Object getFieldValue(Descriptors.FieldDescriptor fd, Object value) {
+        if (fd.getJavaType().equals(Descriptors.FieldDescriptor.JavaType.FLOAT)) {
+            floatCheck(value);
```
Comment (lavkesh) [inline src/main/java/io/odpf/depot/message/proto/ProtoOdpfParsedMessage.java:61]: this should also check for DOUBLE.

**Comment by lavkesh** [comment]
closing this, because the changes are merged http sink branch

**Pull request closed without merging**